### DISCUSSION TITLE
Is ipv6 supported?

### DISCUSSION THREAD

**Discussion by onichandame** [mkckr0/audio-share#47] [Q&A]
My PC is in a cable-only NAT network where the gateway is connected to the upper LAN. My Android phone is in the upper LAN by wifi. Hence my phone cannot connect to the PC behind NAT by ipv4. Both devices have public ipv6 addresses. Is it possible to connect them by ipv6 to bypass the NAT?

**Comment by mkckr0** [comment]
At present, ipv6 is not supported. But I'll add it in future release.

**Comment by mkckr0** [comment]
#48
